pkg/config/config.go

```diff
@@ -112,6 +120,46 @@ func (c *ConfigInfo) writeToFile() error {
 	return nil
 }
 
+// SetConfiguration Set Odo configurations
+func SetConfiguration(component string, value bool) error {
+	cfg, err := New()
+	if err != nil {
+		return errors.Wrap(err, "unable to modify configuration")
+	}
+
+	cfg.OdoConfigurations.UpdateNotification = value
+	err = cfg.writeToFile()
+	if err != nil {
+		return errors.Wrapf(err, "unable to set updatenotification to %t ", value)
+	}
+	return nil
+}
+
+// Getconfiguration get odo configurations
+func GetConfiguration() error {
+	cfg, err := New()
+	if err != nil {
+		return errors.Wrap(err, "unable to get current configuration")
+	}
+
+	bs := []byte(fmt.Sprintf("%v", cfg.OdoConfigurations))
+	if e := yaml.Unmarshal(bs, &cfg.OdoConfigurations.UpdateNotification); e != nil {
+		cfg.OdoConfigurations.UpdateNotification = true
+	}
+	fmt.Printf("%+v\n", cfg.OdoConfigurations)
+	return nil
+}
+
+// GetupdateNotificationStatus returns the value of UpdateNotification config
+func GetupdateNotificationStatus() bool {
```

Comment:
What is the difference between this and `GetConfiguration`?
It looks like they are both doing almost the same.